PROJECT

```diff
@@ -41,7 +41,38 @@ resources:
   controller: true
   domain: nvidia.com
   group: apps
+<<<<<<< HEAD
```

Comment:
unsolicited review: I see several of these rebase-y-looking lines in a few files, FYI.

Reply:
Good reviews always welcome! Thanks for pointing this out, i have fixed the rebase errors.